container-images/cuda/Containerfile

```diff
@@ -1,4 +1,4 @@
-ARG CUDA_VERSION=12.8.1
+ARG CUDA_VERSION=13.0.2
```

Comment:
Looking at https://github.com/ggml-org/llama.cpp/blob/master/ggml/src/ggml-cuda/CMakeLists.txt#L29 I think switching to CUDA 13.0 would drop support for some older GPUs e.g Pascal.

The upstream llama.cpp container images are build on 12.4.0 AFAICT.